### ISSUE TITLE
Add my disqus

### ISSUE THREAD

**Issue by stephenjbarr** [stephenjbarr/stephen-barr-hakyll-bootstap-blog#12]

**Comment by stephenjbarr** [comment]
Shortname: stevejb1

**Comment by stephenjbarr** [comment]
``` html

<div id="disqus_thread"></div>
<script type="text/javascript">
    /* * * CONFIGURATION VARIABLES * * */
    var disqus_shortname = 'stevejb1';

    /* * * DON'T EDIT BELOW THIS LINE * * */
    (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>
```

Adding to the posts template.

**Comment by stephenjbarr** [comment]
Fixed in d19d409
